### PR TITLE
docs: add "Deployment to Shared Hosting Services"

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add "Deployment to Shared Hosting Services"

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/installation/running.rst [hunk 1]
```diff
@@ -426,6 +426,87 @@ Setting Environment
...
+
+The best way is to set the document root to the **public** folder in the server
+configuration::
+
+    └── example.com/ (project folder)
+        └── public/  (document root)
+
+For example, if you use cPanel, see:
+
+- `How to change the document root of the primary domain <https://support.cpanel.net/hc/en-us/articles/360055211213-How-to-change-the-document-root-of-the-primary-domain>`_
```
Comment (datamweb) [inline user_guide_src/source/installation/running.rst:454]: A shared hosting subscriber how can access To WHM as root (step 4)?
Assuming it has access to WHM. Where is the menu he explained(step 5)?

Screenshot 2024-02-17 064944

Comment (kenjis) [reply user_guide_src/source/installation/running.rst:454]: See https://docs.cpanel.net/knowledge-base/accounts/how-to-log-in-to-your-server-or-account/#how-to-access-whm

Comment (kenjis) [reply user_guide_src/source/installation/running.rst:454]: How do I log in to my cPanel account as root? – cPanel
https://support.cpanel.net/hc/en-us/articles/360051754554-How-do-I-log-in-to-my-cPanel-account-as-root

Comment (kenjis) [reply user_guide_src/source/installation/running.rst:454]: Perhaps we are not normally allowed to log in as root. If so, this method is not generally available. 
In that case, it should not be mentioned in the documentation.

Comment (datamweb) [reply user_guide_src/source/installation/running.rst:454]: I had a call with the hosting company.
The possibility of logging for shared hosting users is provided only  through cPanel NOT WHM.

Regarding the change of Document Root, Answer was, considering that the host is shared, it is **NOT** possible to change the Document Root.

Screenshot 2024-02-17 081936

Comment (kenjis) [reply user_guide_src/source/installation/running.rst]: @datamweb Thank you for investigation! I've removed the URLs for cPanel.

File: user_guide_src/source/installation/running.rst [hunk 2]
```diff
@@ -426,6 +426,85 @@ Setting Environment
...
+The best way is to set the document root to the **public** folder in the server
+configuration::
+
+    └── example.com/ (project folder)
+        └── public/  (document root)
+
+Check with your hosting service provider to see if you can change the document root.
+Unfortunately, if you cannot change the document root, go to the next way.
+
+Using Two Directories
```
Comment (datamweb) [inline user_guide_src/source/installation/running.rst:455]: Well, I tested this method in practice, it works well.
@kenjis, thank you for following up.

**Review by datamweb — APPROVED** [review]

File: user_guide_src/source/installation/running.rst (hunk 2 above)
Comment (kenjis) [reply user_guide_src/source/installation/running.rst:455]: This method (and a few hacks) worked on 000webhostapp.
https://kenjis.000webhostapp.com/

Comment (datamweb) [reply user_guide_src/source/installation/running.rst:455]: Oh!! In 2022, someone asked me this question in an email :innocent: 

Screenshot 2024-02-20 131713

**Pull request merged**